input/pagecontent/payload.md

```diff
@@ -1,7 +1,106 @@
 
-### Genomic Report Structure
+As shown in Figure 1 below, the Genomic payload has multiple resources, the principle ones specific to genomics being GenomicReport, Variant, plus those that are closely tied such as DiagnosticImplications and TherapeuticImplications.
 
-**Insert diagram of the general report structure. Reference GRIG FHIR IG for examples.**
+<figure>
+    <object data="gdx_profiles.svg" type="image/svg+xml"></object>
+    <figcaption>Figure 1. Genomic Data Profiles</figcaption>
+</figure>
 
-#### Bundle types
-*Insert options of bundle options and dynamic searchset options*
+
+### Payload Development Process
+The Data Exchange use case worked across the data creating member to develop the initial consensus for genomic report information that would be included in Phase 0, the initial implementation of the GRIG. To understand what data required FHIR representation, the members provided the basic data elements  that needed to be delivered as part of their existing genomic test results. Along with the data elements, metadata about each element were also captured, including any required elements, data element code system(s), and code systems that can be implemented now or in the future. 
+
+Once the data elements were catalogued, the members mapped them to an appropriate set of FHIR resources based on FHIR R4, the GRIG STU2, and mCODE STU3. The results were compared across the report producers to determine the overlapping set of FHIR resources. Those that were common or closely aligned between the report producers were agreed upon as the initial set for this IG.
+
+These data elements were the focus of Phase 0 and will represent a starting point for this IG. 
+
+For additional context, samples for a variety of report types, such as general genomic reporting, somatic reporting, variant reporting, and PGx reporting are available under the [How to Use This Guide](http://hl7.org/fhir/uv/genomics-reporting/STU2/#how-to-use-this-guide) section of the [Genomics Reporting IG (STU2)](http://hl7.org/fhir/uv/genomics-reporting/STU2).
+
+### Outcomes 
+The major outcomes of the payload development process are represented by the constraints on the FHIR resources in this IG and shown in the profiles. In addition, there are other design decisions that are shown below to help guide the Data Exchange community through early implementation of FHIR in their workflows. 
+
+#### Decision: Bundle types
+There are a number of options for the FHIR bundle type that may be used to deliver the genomic test results. The intention is to provide options to the report producers as they increase their FHIR capabilities. The decision of how to generate the FHIR bundle must take into consideration the [transport mechanism](transport.md) used. 
```

Comment:
Looks like the "transport.md" link needs to be changed to "transport.html". Sorry I missed that earlier.